internal/tfimport/tfimport.go

```diff
@@ -466,12 +472,17 @@ func (ir Resource) ImportID(interactive bool) (string, error) {
 
 // Importable returns an importable Resource which contains an Importer, and whether it successfully created that resource.
 // pcv represents provider config values, which will be used if the resource does not have values defined.
-func Importable(rc terraform.ResourceChange, pcv importer.ConfigMap) (*Resource, bool) {
+func Importable(rc terraform.ResourceChange, pcv importer.ConfigMap, interactive bool) (*Resource, bool) {
 	ri, ok := Importers[rc.Kind]
 	if !ok {
 		return nil, false
 	}
 
+	if _, reqInteractive := RequiresInteractive[rc.Kind]; reqInteractive && !interactive {
+		fmt.Printf("Martin found %v requires interactive\n", rc.Kind)
```

Comment:
Martin found ? :)

Reply:
Ah yes. I purposely put my name in to make it glaringly obvious when I forget to remove log messages :)